apps/lfx-one/src/app/modules/profile/components/profile-edit-drawer/profile-edit-drawer.component.html

```
@@ -0,0 +1,345 @@
+<!-- Copyright The Linux Foundation and each contributor to LFX. -->
+<!-- SPDX-License-Identifier: MIT -->
+
+<!-- Profile & Account edit drawer (LFXV2-2742) — replaces the former edit dialog. -->
+<p-drawer
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)